src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -35,12 +38,31 @@ attributes:
  persistVolumes: ‘false’
 ----
 
-.Asynchronous storage
+=== Comparison table between I/O of  Ephemeral (`emptyDir`) vs  Persitent (`EBS` more about link:https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs[AWS EBS])
+
+[cols="3", options="header"]
+|===
+|Command
+|Ephemeral
+|Persitent
+
+|Clone  Eclipse Che
+|0m 19s
+|1m 26s
+
+|Generate 1000 random files
```

Comment:
Looks interesting - how 1000 random files were generated for the synthetic test

Reply:
```
for n in {1..1000}; do
    dd if=/dev/urandom of=file$( printf %03d "$n" ) bs=1 count=$(( RANDOM + 1024 ))
done
```